include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3370,6 +3370,11 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
                                            *this->dof_info);
     }
 
+  const bool is_ecl =
+    this->dof_access_index ==
+      internal::MatrixFreeFunctions::DoFInfo::dof_access_cell &&
+    this->is_interior_face() == false;
```

Comment:
Can we give the variable a better, name, e.g.
```suggestion
  const bool accesses_exterior_dofs =
    this->dof_access_index ==
      internal::MatrixFreeFunctions::DoFInfo::dof_access_cell &&
    this->is_interior_face() == false;
```